src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/ImplementIDisposableCorrectlyTests.cs

```diff
@@ -519,7 +519,46 @@ public override void Dispose()
 
         #region CSharp FinalizeOverride Unit Tests
 
-        [Fact]
+        [Fact, WorkItem(1950, "https://github.com/dotnet/roslyn-analyzers/issues/1950")]
+        public void CSharp_CA1063_FinalizeOverride_Diagnostic_SimpleFinalizeOverride_OverridesDisposeBool()
```

Comment:
❗️ This test is not correct. One of the following alterations must be made to the test input before the test can be considered correct:

1. Remove the finalizer from `B`
2. Remove the finalizer from `C`

Failure to address this will result in *both* finalizers getting called, which is not desirable.